### PR TITLE
Tour managment/general - Env, Redux and @material-ui changes.

### PR DESCRIPTION
**Pull request by igortokhunts**
The changes.
1. Env - Removed env files and added the .env.example file.
2. Redux - Initial setup(included redux-devtools) with an example files(action, reducer).
3. Material - Uninstall @material-ui v4
4. Commented the Context.Api files for future deletion.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+REACT_APP_SERVER_URL=http://localhost:5001
+REACT_APP_BASE_URL=http://localhost:5001
+REACT_APP_AUTH_URL=http://localhost:5001/auth
+REACT_APP_NOTIFICATION_URL=http://localhost:5001/notification
+REACT_APP_FILE_SERVICE_URL=http://localhost:5001/files
```
Comment (mirztate1-flutterint) [inline .env.example:5]: add new extra line at the end please

File: src/store/reducers/index.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from "redux";
+import example from "./example.reducer";
+
+
+export const rootReducer = combineReducers({
+    example,
+});
```
Comment (mirztate1-flutterint) [inline src/store/reducers/index.js:7]: the same here (extra line)

File: src/store/actions/example.actions.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import EXAMPLE_TYPES from '../types/example.types';
+
+export const example = payload => dispatch => {
+    dispatch({type: EXAMPLE_TYPES.INIT_APPLICATION_EXAMPLE, payload});
+};
```
Comment (mirztate1-flutterint) [inline src/store/actions/example.actions.js:5]: here as well

File: src/store/types/createPrefixe.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+/**
+ * @description - Returns a function which know provided prefix, that function will return string prefix + name.
+ * @param {String} prefix - prefix
+ * @returns {Function}
+ */
+export default prefix => name => `@@APP/${prefix}/${name}`;
```
Comment (mirztate1-flutterint) [inline src/store/types/createPrefixe.js:6]: the same here (extra line)

File: src/store/types/example.types.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import createPrefix from './createPrefixe';
+
+const createExamplePrefix = createPrefix('EXAMPLE');
+
+const EXAMPLE_TYPES = {
+    INIT_APPLICATION_EXAMPLE: createExamplePrefix('INIT_APPLICATION_EXAMPLE'),
+};
+
+export default EXAMPLE_TYPES;
```
Comment (mirztate1-flutterint) [inline src/store/types/example.types.js:9]: the same here (extra line)

File: .env.example (hunk 1 above)
Comment (sasuntitan) [reply .env.example:5]: is there linter error?

Comment (igortoghunts) [reply .env.example:5]: there is no @sasuntitan jan, need we add some lint configs included the one for this case or we are good with ignoring extra lines?

**Review by sasuntitan — APPROVED** [review]

**Pull request merged**